app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+            need to allow them specifically. For people that use Android 7+, you need to use Emulators like Mumu or Nox which usually run older
+            Android versions. Device with Android 6 or lower also work. iOS does not have this limitation and works fine in all versions if you do the
+            additional steps. Using BlueStacks is not reconmended but may be possible, although the last confirmed working verions for this is 0.0.29
+            which doesn't support the needed HTTPS functionality. This also requires root access. We are not liable for any form of damage caused by
+            using any BlueStacks setup guide.
+          </Accordion.Content>
+
+          {/* Documents & programs */}
+          <Accordion.Title active={activeIndex === 1} index={1} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Documents & programs
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 1}>
+            <p>
+              <ul>
+                {/* <li>
+                  <a href="https://www.reddit.com/r/summonerswar/comments/l43ueg/making_swex_work_in_2021/" target="_blank">
+                    Making Swex work in 2021
+                  </a>
+                </li> */}
+                <li>
+                  <a href="https://www.mumuglobal.com/" target="_blank">
+                    Mumu for Windows{' '}
+                  </a>
+                  , make sure to download the 64 bit verision under the colored button.
+                </li>
+                <li>
+                  <a href="https://adl.netease.com/d/g/a11/c/mac" target="_blank">
+                    Mumu for Mac OS{' '}
+                  </a>
+                  (Not yet compatible with M1).
+                </li>
+                <li>
+                  <a href="https://www.bignox.com/en/download/fullPackage/win_64?beta" target="_blank">
+                    Nox app player (64-bit) Windows.{' '}
+                  </a>
+                </li>
+                <li>
+                  <a href="https://www.bignox.com/en/download/fullPackage/mac_fullzip?beta" target="_blank">
+                    Nox app player (64-bit) MacOS.{' '}
+                  </a>
+                </li>
+                <li>
+                  <a href="https://drive.google.com/file/d/1mQuivBo2lpRvPI4Obapvcb9NYDcaSUoh/view" target="_blank">
+                    Mumu setup guide{' '}
+                  </a>
+                </li>
+              </ul>
+            </p>
+          </Accordion.Content>
+
+          {/* mumu setup */}
+          <Accordion.Title active={activeIndex === 2} index={2} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            Mumu Setup
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 2}>
+            <p>
```

Comment:
Its usually not great practise to wrap a list in a p element. I suggest to just remove the <p\> element and use solely the <ul\> (or other HTML elements)